src/modules/dashboard/components/AwardDialog/AwardDialog.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useCallback, useState } from 'react';
+import { FormikProps } from 'formik';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+import { defineMessages } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import Decimal from 'decimal.js';
+
+import Dialog from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+
+import { DEFAULT_TOKEN_DECIMALS } from '~constants';
+import { ActionTypes } from '~redux/index';
+import { pipe, withMeta, mapPayload } from '~utils/actions';
+
+import DialogForm, {
+  AwardAndSmiteDialogProps,
+  AwardAndSmiteDialogFormValues,
+  AwardAndSmiteFormValidationSchema,
+} from '../AwardAndSmiteDialogForm';
```

Comment:
All of `Smite`, `Award` and `AwardAndSmite` componets should live in the same folder.

Something like `src/modules/dashboard/components/AwardAndSmiteDialogs` or something...

Please move them to a single location